ioc/start.sh

```diff
@@ -156,12 +155,17 @@ fi
 
 # Launch the IOC ***************************************************************
 
-if [[ ${TARGET_ARCHITECTURE} == "rtems" ]] ; then
-    echo "RTEMS IOC startup - copying IOC to RTEMS mount point ..."
-    cp -r ${IOC} ${K8S_IOC_ROOT}
-    sleep 100
-else
+if [[ ${TARGET_ARCHITECTURE} == "linux-x86_64" ]] ; then
```

Comment:
@coretl in fact this is not required because the override start.sh is called with exec. i.e. completely takes over the process and will not come back to the original start.sh

Reply:
It might be clearer to the reader to be explicit so that they don't have to know that, and then it also matches the `bash ...` statements (which I think does not do the same and further lines do get run?).